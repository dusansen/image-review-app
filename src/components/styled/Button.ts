import styled from 'styled-components';
import ButtonProps from '../../models/Button.props';

export const Button = styled.button<ButtonProps>`
  background-color: ${props => props.buttonType === 'like' ? '#273c75' : '#353b48'};
  color: #fff;
  height: min-content;
  padding: 0.8rem 1.2rem;
  border-radius: 100px;
  border: none;
  cursor: pointer;
  min-width: 6rem;
`;