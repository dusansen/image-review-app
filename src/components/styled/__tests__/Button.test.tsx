import renderer from 'react-test-renderer';
import { Button } from '../Button';

describe('Button', () => {
  it('should render Button with styles', () => {
    const tree = renderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
