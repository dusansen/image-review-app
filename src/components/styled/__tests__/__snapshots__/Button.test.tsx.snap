// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render Button with styles 1`] = `
<button
  className="sc-bdvvtL kBliwQ"
/>
`;
